src/main.cpp

```diff
@@ -510,19 +536,19 @@ void tick_all_interfaces(const SysTick_s &current_system_tick)
             int(fsm.get_state()),
             buzzer.buzzer_is_on(),
             drivetrain.drivetrain_error_occured(),
-            torque_controller_mux.getTorqueLimit(),
+            torque_controller_mux.get_tc_mux_status().current_torque_limit_enum,
```

Comment:
What is current_torque_limit_enum? What does "enum" mean in this context? Is there a more self-explanatory term we can use

Reply:
it means this:

```cpp

enum class TorqueLimit_e
{
    TCMUX_FULL_TORQUE = 0,
    TCMUX_MID_TORQUE = 1,
    TCMUX_LOW_TORQUE = 2,
    TCMUX_NUM_TORQUE_LIMITS = 3,
};

struct TorqueControllerMuxStatus
{
    TorqueControllerMuxError current_error;
    ControllerMode_e current_controller_mode_;
    TorqueLimit_e current_torque_limit_enum;
    float current_torque_limit_value;
    bool output_is_bypassing_limits;
};
```

it refers to the current torque limit we are at, whether it be full, "mid" or "low" torque as of right now.